engine/schema/src/main/java/com/cloud/upgrade/SystemVmTemplateRegistration.java

```diff
@@ -108,7 +109,10 @@ public class SystemVmTemplateRegistration {
     private static Integer LINUX_12_ID = 363;
     private static final Integer SCRIPT_TIMEOUT = 1800000;
     private static final Integer LOCK_WAIT_TIMEOUT = 1200;
+    protected static final String TEMPLATES_DOWNLOAD_REPOSITORY_KEY = "downloadurl";
```

Comment:
```suggestion
    protected static final String TEMPLATES_DEFAULT_DOWNLOAD_REPOSITORY_KEY = "downloadurl";
```

or _TEMPLATES_DOWNLOAD_URL_KEY_